docs/source/faq.rst

```diff
@@ -77,3 +77,23 @@ If you want to save and resume studies,  it's handy to use SQLite as the local s
     study.optimize(objective)  # The state of `study` will be persisted to the local SQLite file.
 
 Please see :ref:`rdb` for more details.
+
+
+How to change logging levels of Optuna?
```

Comment:
Maybe something like  `How to suppress log messages of Optuna?` might be more direct?